storage.go

```diff
@@ -22,6 +22,35 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 	return o
 }
 
+func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
+	rp := s.getAbsPath(path)
+
+	var nextPos int64 = 0
+	isExist, err := s.bucket.IsObjectExist(rp)
+	if err != nil {
+		return
+	}
+
+	if isExist {
+		props, errGetMeta := s.bucket.GetObjectDetailedMeta(rp)
```

Comment:
In `createAppend`, we can use `AppendObject` with position 0 and size 0 to create an Object.

Reply:
Does go-service-oss need to support append to an existing object?

Reply:
Yes, but not in this way.

`CreateAppend` needs to `create` an append object and should not do other jobs.

In order to append to an existing object, use can:

- Use `Stat` to get an append object.
- Use `Create` to create an append object if they know the offset.